src/plugins/data-inview/data-inview-en.hbs

```diff
@@ -42,11 +39,11 @@
 		</details>
 	</section>
 
-	<p>Different example text. Different example text. Different example text. Different example text. Different example text. Different example text. Different example text. Different example text.</p>
+	<p role="presentation">Different example text. Different example text. Different example text. Different example text. Different example text. Different example text. Different example text. Different example text.</p>
```

Comment:
It's clear that filler text is purely decorative and do not provide any useful information. In fact, it is just occupying some space to provide an user experience on how the plugin work.